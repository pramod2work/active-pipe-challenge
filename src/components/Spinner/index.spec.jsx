import React from 'react'
import { shallow } from 'enzyme'

import { Spinner } from './index'

describe('Spinner Component test', () => {
  test('Component Snapshot Testing', () => {
    const component = shallow(<Spinner />)
    expect(component).toMatchSnapshot()
  })
})