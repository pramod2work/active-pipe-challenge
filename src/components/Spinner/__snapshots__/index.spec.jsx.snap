// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Spinner Component test Component Snapshot Testing 1`] = `
<div>
  <img
    alt="spinner"
    className="spinner"
    src="spinner.gif"
  />
</div>
`;
